packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -1089,9 +1089,19 @@ mixin WidgetInspectorService {
       name: 'getRootWidgetSummaryTree',
       callback: _getRootWidgetSummaryTree,
     );
-    _registerServiceExtensionWithArg(
+    registerServiceExtension(
       name: 'getDetailsSubtree',
-      callback: _getDetailsSubtree,
+      callback: (Map<String, String> parameters) async {
+        assert(parameters.containsKey('objectGroup'));
+        final int subtreeDepth = int.parse(parameters['subtreeDepth'] ?? '2');
+        return <String, Object>{
+          'result': _getDetailsSubtree(
+              parameters['arg'],
```

Comment:
nit: fix indentation. should be 2 not 4.

Reply:
done